CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## v1.4.0
 
 * Fixed an issue when feedback UI was always appearing for short routes ([#2871](https://github.com/mapbox/mapbox-navigation-ios/pull/2871))
+* `NavigationViewController.styleManager` is now publicly scoped allowing for programmatic style changes ([#2888](https://github.com/mapbox/mapbox-navigation-ios/pull/2888))
```

Comment:
From the developer’s point of view, this property never existed before, so we can talk about it as an addition:

```suggestion
* Added the `NavigationViewController.styleManager`, `StyleManager.currentStyleType`, and `StyleManager.currentStyle` properties and the `StyleManager.applyStyle(type:)` method to manually change the UI style at any time. ([#2888](https://github.com/mapbox/mapbox-navigation-ios/pull/2888))
```